MFVNeutralino/plugins/TrackHistos.cc

```diff
@@ -102,7 +102,7 @@ void MFVTrackHistos::analyze(const edm::Event& event, const edm::EventSetup&) {
   double bsz = mevent->bsz;
   const math::XYZPoint bs(bsx, bsy, bsz);
 
-  const int track_rescaler_which = jmt::TrackRescaler::w_JetHT; // JMTBAD which rescaling if ever a different one
+  const int track_rescaler_which = jmt::TrackRescaler::w_BTagDispJet; // JMTBAD which rescaling if ever a different one
```

Comment:
This should be configurable rather than hardcoded, given our current use of these things.